### PR TITLE
updated content, blur background

### PR DESCRIPTION
**Pull request by squiddot**
## What

Updating infosite content, blur background, remove index accordions

## Link to Issue
No current issue, just needs updating

### Steps for testing

clone site, run dev command, might have issues with node version

### Images

Screen Shot 2023-09-12 at 1 57 40 PM

### CODE REVIEW

**Review by buffet-time — APPROVED** [review]

File: pages/contribute.vue [hunk 1]
```diff
@@ -1,60 +1,87 @@
...
-          <a href="https://github.com/leaderboardsgg/leaderboard-site/issues">Issues</a>
+          <a href="https://github.com/leaderboardsgg/leaderboard-site/issues"
+            >Issues</a
+          >
         </li>
       </ul>
 
       <h3>Back End</h3>
       <ul>
+        <li>Technologies: C#, ASP.NET Core, .NET 6.0, PostgreSQL</li>
```
Comment (erunks) [inline pages/contribute.vue]: I thought we were on .NET 7.0? That's at least what it says in the [backend readme](https://github.com/leaderboardsgg/leaderboard-backend#visual-studio-codeother-editors)

File: pages/contribute.vue [hunk 2]
```diff
@@ -1,60 +1,87 @@
...
-    <HeroBanner color="blue"><h1><span>How You Can Help</span></h1></HeroBanner>
+    <HeroBanner color="blue"
+      ><h1><span>How You Can Help</span></h1></HeroBanner
+    >
     <h3 class="intro">Contributor Guidelines</h3>
-    <div>
-      <h2>Thank you for your interest in contributing to LeaderboardsGG. We're glad you found your way here, and we're always in need of new contributors.</h2>
+    <div class="container">
+      <h2>
+        Thank you for your interest in contributing to LeaderboardsGG. We're
```
Comment (erunks) [inline pages/contribute.vue]: We reference the project here as `LeaderboardsGG`, but elsewhere as `Leaderboards.gg` or `leaderboards.gg`. We should make this is consistent, whichever way we want to go with it

File: pages/index.vue [hunk 3]
```diff
@@ -7,30 +7,188 @@
...
+        prompted us to create a website that would be proactive in providing
+        these critical components of a user-focused website.
+      </p>
+      <h3 id="our-promise">Our Promise</h3>
+      <p>
+        In order to best serve our users, leaderboards.gg will remain free and
+        open source, ad-free, and without paid features. This means that all the
+        features of the website will be available to all users free of charge,
+        and we will never put features behind a paywall. As a registered Open
+        Collective, Leaderboards.gg is funded entirely by donations, all of
```
Comment (erunks) [inline pages/index.vue]: Would we want to link our open collective here?

**Review by erunks — COMMENTED** [review]
A couple of thoughts, but otherwise looking good

File: pages/contribute.vue (hunk 1 above)
Comment (TheTedder) [reply pages/contribute.vue]: Yes, we've upgraded. Not that there's a huge difference.

Comment (squiddot) [reply pages/contribute.vue]: Updated

File: pages/contribute.vue (hunk 2 above)
Comment (squiddot) [reply pages/contribute.vue]: Changed the few I could find. I think thats all of them?

**Review by erunks — APPROVED** [review]

**Pull request merged**